ui/accordion.js

```diff
@@ -499,7 +502,7 @@ return $.widget( "ui.accordion", {
 			});
 		} else if ( toShow.length ) {
 			this.headers.filter(function() {
-				return $( this ).attr( "tabIndex" ) === 0;
+				return $( this ).attr( "tabIndex" ) === "0";
```

Comment:
AFAIK IE8 is fine here; IE<8 have attroperties and they return numbers but you don't support IE<8 so it should be good.